back/app/services/admin_publications_filtering_service.rb

```diff
@@ -101,6 +101,16 @@ def for_homepage_filter(scope)
     scope.where.not(id: unmoderated_parents)
   end
 
+  # This filter excludes AdminPublications that are for projects in folders that are still included in the scope.
+  # This is used by the BO 'your projects' list to avoid duplicates when a moderated project is in a moderated folder.
+  add_filter('exclude_projects_in_included_folders') do |scope, options|
+    next scope unless ['true', true, '1'].include? options[:exclude_projects_in_included_folders]
```

Comment:
Will this sometimes be string, boolean etc depending on where it's called from. Slightly unsatisfactory bit of code, but if it's needed so be it!

Reply:
This is our convention. I believe it was considered a more robust check. Maybe a good thing for our next CDM?

Reply:
Added to CDM [here](https://www.notion.so/govocal/Back-end-Code-Discussion-Meeting-BE-CDM-a94ef3cfdd9849e4bde22d2b10dcffeb?pvs=4#afdb04ee63ab492489b8fdcab6cabd17)